projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-theme.scss

```diff
@@ -369,9 +369,9 @@
     );
 
     $grouparea-padding: (
-        comfortable: rem(16px) rem(24px),
-        cosy: rem(8px) rem(16px),
-        compact: rem(4px) rem(12px)
+        comfortable: rem(16px) rem(40px),
+        cosy: rem(8px) rem(24px),
+        compact: rem(4px) rem(16px)
     );
 
```

Comment:
This breaks the compact and cosy inheritance between grid elements.

Reply:
Ok, in that case, is it a better idea to close to pull request and assign the issue to you ?

Reply:
I don't see how this is a styling issue. from what I see the gap under the grouped rows is caused by incorrect calc on grid body. Try to add one more chip and the problem disappear. Can you explain why do you think it's caused by CSS/SASS

Reply:
Actually, this is caused because when you add the last chip there is an arrow indicator that points to the right and there is no space just for a millisecond. You can see a small jumping of that whole area caused by that arrow.

https://drive.google.com/file/d/1iudzzRe-VY9mzvzb8nGMdhIcm0BWBoxF/view

Reply:
I'll investigate it further then, thought it's a css issue, because adding extra padding resolves it.